geometry_msgs/Pose[] north_west_goal
geometry_msgs/Pose[] north_goal
geometry_msgs/Pose[] north_east_goal
geometry_msgs/Pose[] east_goal
geometry_msgs/Pose[] west_goal
geometry_msgs/Pose[] south_west_goal
geometry_msgs/Pose[] south_goal
geometry_msgs/Pose[] south_east_goal
geometry_msgs/Pose[] middle_goal

string[] name_north_west_goal
string[] name_north_goal
string[] name_north_east_goal
string[] name_east_goal
string[] name_west_goal
string[] name_south_west_goal
string[] name_south_goal
string[] name_south_east_goal
string[] name_middle_goal

string[] identity_north_west_goal
string[] identity_north_goal
string[] identity_north_east_goal
string[] identity_east_goal
string[] identity_west_goal
string[] identity_south_west_goal
string[] identity_south_goal
string[] identity_south_east_goal
string[] identity_middle_goal
